### ISSUE TITLE
Premake script to just compile

### ISSUE THREAD

**Issue by holychicken99** [premake/premake-core#1840]
Is there to way to ask Premake to just compile files and not link them ?
The usual configuration compiles aswell as links them .

**Comment by samsinsane** [comment]
I don't believe this functionality exists, I'm not sure how that would work for VS or XCode, can either of them be instructed to not link?

**Comment by nickclark2016** [comment]
I'm not aware of it for VS.  I know for the CLI, you can specify `/C` as an arg to `CL.exe`, but I don't know if that's supported in the IDE.  Not familiar enough with XCode to make an educated statement.
